src/pages/history/index.tsx

```
@@ -1,74 +1,73 @@
-                {dayjs(conversation.createdAt).format(TIME_FORMAT)}
-              </p>
-              <div className="card-actions justify-end mt-2">
+      <div className="p-4 flex justify-center">
+        <ul className="list bg-base-200 rounded-box w-xl">
+          {conversationList && conversationList.length <= 0 && (
+            <li className="list-row flex text-center">
+              <p className="w-full">No conversation was found.</p>
+            </li>
+          )}
+          {conversationList?.map((conversation) => (
+            <li className="list-row flex">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)